src/neptune_scale/api/run.py

```diff
@@ -90,21 +84,21 @@ def __init__(
         self,
         *,
         run_id: str,
-        project: Optional[str] = None,
-        api_token: Optional[str] = None,
+        project: str | None = None,
```

Comment:
didn't | syntax for type appear in python 3.10? at least that's what https://docs.python.org/3/library/typing.html#typing.Union says. 
It should have failed in some tests on 3.9

Reply:
Seems like it should be so, however both mypy, and python 3.9 interpreter don't have issues with importing the code, which is quite weird, but I don't have the space to dig into this deeper -- I'll just go back to Optional